### ISSUE TITLE
Request: An option to disable notification bubble

### ISSUE THREAD

**Issue by GeorgeWL** [mozilla/contain-facebook#717]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when facebook container shows a notification

**Describe the solution you'd like**
An option to disable notification bubble

**Describe alternatives you've considered**
Remove notification bubble

![image](https://user-images.githubusercontent.com/5931248/105395990-46659e00-5c17-11eb-8bf5-9e59966bbebc.png)

**Comment by pjonastik** [comment]
I agree that it is a quite annoying when that the notification bubble constantly blinking when I read an article. I had to remove it from Toolbar.
